### ISSUE TITLE
[DN3Doom & FlexiHUD] Crash after drop items.

### ISSUE THREAD

**Issue by SkullGamer205** [jekyllgrim/UniversalFlexibleHUD#4]
**Engine:** GZDoom
**Engine Version:** g4.12pre-88-g61c2f58c-m
**OS:** Arch Linux
**OS Version:** 6.5.9-arch2-1

Mod Link: https://www.moddb.com/mods/dn3doom/downloads/dn3doom-v1-07

Errors:
```
Called from Inventory.PrevInv at gzdoom.pk3:zscript/actors/inventory/ inventory.zs, line 1169
Called from JGPUFH_FlexibleHUD.NextItem at FlexiHUD:ufhzscript/ufh_hud.zs, line 3252
Called from JGPUFH_FlexibleHUD.DrawInventoryBar at FlexiHUD:ufhzscript/ufh_hud.zs, line 3127
Called from JGPUFH_FlexibleHUD.Draw at FlexiHUD:ufhzscript/ufh_hud.zs, line 314
```

Preview:
![изображение](https://github.com/jekyllgrim/UniversalFlexibleHUD/assets/61167080/06aae816-ecaf-40d7-b635-c85b1ef1c1c6)

https://github.com/jekyllgrim/UniversalFlexibleHUD/assets/61167080/19fa12eb-9389-4f13-8fb2-c7d16a47c16a

**Comment by jekyllgrim** [comment]
Thanks for the report. This wasn't specified to DN3Doom, it could be triggered in any game/mod with an inventory bar. This is fixed in 64ef52511a281ce1219f7dbf1cc134ad3ba72683.

**Comment by SkullGamer205** [comment]
You're welcome. 
 And thank you for correcting the error. 
 I wish you and your project all the best.
